api/v1alpha1/database_types.go

```diff
@@ -208,12 +208,19 @@ type TaskInfoSummaryResponse struct {
 
 // +kubebuilder:object:generate:=false
 type ProfileResponse struct {
-	Id              string `json:"id"`
-	Name            string `json:"name"`
-	Type            string `json:"type"`
-	EngineType      string `json:"engineType"`
-	LatestVersionId string `json:"latestVersionId"`
-	Topology        string `json:"topology"`
+	Id              string    `json:"id"`
+	Name            string    `json:"name"`
+	Type            string    `json:"type"`
+	EngineType      string    `json:"engineType"`
+	LatestVersionId string    `json:"latestVersionId"`
+	Topology        string    `json:"topology"`
+	Versions        []Version `json:"versions"`
+}
+
+type Version struct {
+	Id          string `json:"id"`
+	Name        string `json:"name"`
+	Description string `json:"description"`
```

Comment:
We don't require the version anymore. We'll resolve the profiles either by name and/or by id, giving preference to name if both are provided.